<ui version="4.0" >
 <class>CaptureDialog</class>
 <widget class="QDialog" name="CaptureDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Capture data - Veusz</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>&amp;Datasets:</string>
       </property>
       <property name="buddy" >
        <cstring>descriptorEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="descriptorEdit" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Enter a descriptor to describe the format of the incoming data, e.g. "x,+,- y,+-" (see the Data->Import dialog box for details)</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="captureGroup" >
     <property name="title" >
      <string>Capture method</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QRadioButton" name="captureFileButton" >
        <property name="text" >
         <string>&amp;File or named pipe</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>&amp;Filename:</string>
          </property>
          <property name="buddy" >
           <cstring>filenameEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="filenameEdit" />
        </item>
        <item>
         <widget class="QPushButton" name="filenameButton" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="captureInternetButton" >
        <property name="text" >
         <string>Connect to &amp;Internet TCP socket</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>&amp;Host:</string>
          </property>
          <property name="buddy" >
           <cstring>hostEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="hostEdit" >
          <property name="text" >
           <string>localhost</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="portEdit" >
          <property name="text" >
           <string>10000</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="captureProgramButton" >
        <property name="text" >
         <string>E&amp;xternal program</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" >
        <item>
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>Command line:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="commandLineEdit" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="stopGroup" >
     <property name="title" >
      <string>Stop after</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QRadioButton" name="clickingStopButton" >
        <property name="text" >
         <string>Clicking stop button</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="numPtsStopEdit" >
        <property name="text" >
         <string>100</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="numPtsStopButton" >
        <property name="text" >
         <string>Number of datapoints</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QRadioButton" name="timeStopButton" >
        <property name="text" >
         <string>Time period (s)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="timeStopEdit" >
        <property name="text" >
         <string>60</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="captureButton" >
       <property name="text" >
        <string>Ca&amp;pture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CaptureDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>319</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel" >
     <x>180</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
